docs/framework/dataframe.mdx

```diff
@@ -2,97 +2,213 @@
 title: "Dataframe"
 ---
 
-**writer framework places the dataframe at the core of the application**. This is a great way for modeling a complex and massive data system.
-it offers components as `dataframe` to manipulate dataframes. These components allow you to visualize and interact with dataframes.
-
-| compatibility      | dataframe                             |
-|--------------------|---------------------------------------|
-| `pandas.DataFrame` | x                                     |
-| `polar.DataFrame`  | x                                     |
-| `list of records`  | x (with `EditableDataframe`)          |
-
-## Use a dataframe
-
-**a dataframe is simply added to the state**. A component like `dataframe` will be able to display it.
+**Writer Framework has two objects with the name _DataFrame_:**
```

Comment:
Can we add an intro sentence here to help people understand what they're trying to accomplish with dataframes?